src/game/game/game_cutscene.c

```diff
@@ -69,6 +78,36 @@ int32_t Game_Cutscene_Start(int32_t level_num)
     return GF_NOP;
 }
 
+void Game_Cutscene_InitialiseHair(int32_t level_num)
+{
+    if (!g_Config.enable_braid || !g_Objects[O_HAIR].loaded) {
+        return;
+    }
+
+    GAME_OBJECT_ID lara_type = g_GameFlow.levels[level_num].lara_type;
+    if (lara_type == O_LARA) {
+        return;
+    }
+
+    for (int i = 0; i < g_LevelItemCount; i++) {
+        if (g_Items[i].object_number != lara_type) {
+            continue;
+        }
```

Comment:
Can we change it to
```c
ITEM_INFO *lara = NULL;
for (int i = 0; i < g_LevelItemCount; i++) {
    if (g_Items[i].object_number == lara_type) {
        lara = &g_Items[i];
        break;
    }
}
if (!lara) {
    return;
}

do_stuff(lara);
```

Reply:
Slight variation on the proposed change, as we need to pass the item number to `Lara_InitialiseLoad` rather than the item pointer. But regardless, the loop is now exited swiftly.